### PR TITLE
[TASK] Align with new TYPO3 documentation standards

### PR DESCRIPTION
**Pull request by alexander-nitsche**
Relates: https://github.com/TYPO3-Documentation/T3DocTeam/issues/182

### CODE REVIEW

**Comment by alexander-nitsche** [comment]
What is missing: 
(1) Registering this documentation for rendering and publishing at docs.typo3.org with the [webhook](https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingDocForExtension/Webhook.html#github).
(2) Remove any documentation URL from the "External manual" field of the TER extension page configuration at https://extensions.typo3.org as the documentation at docs.typo3.org is detected automatically and a specific URL is error-prone.
(3) Insert composer package name "friendsoftypo3/mediace" into "Composer name of extension" field of the TER extension page configuration.

**Comment by alexander-nitsche** [comment]
How it looks:
Screen Shot 2022-04-04 at 12 44 41

**Comment by alexander-nitsche** [comment]
Added two new commits regarding badges in README and additional sources (TER, repository and docs.typo3.org) in composer.json.

Relates: TYPO3-Documentation/T3DocTeam#185

File: Documentation/Contribution/Index.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+The extension is published under GPL v2+, all included third-party libraries are
+published under their respective licenses.
+
+Authors
+=======
+
+Over 100 contributors have been working on this area while this functionality
+was part of the TYPO3 Core. This package is now maintained by a loose group of
+TYPO3 enthusiasts inside the TYPO3 Community. Feel free to contact `Benni Mack`_
```
Comment (ohader) [inline Documentation/Contribution/Index.rst]: Please avoid pointing to Benni here, refer to the issue tracker instead...

**Review by ohader — CHANGES_REQUESTED** [review]
Avoid pointing to people directly

File: Documentation/Contribution/Index.rst (hunk 1 above)
Comment (alexander-nitsche) [reply Documentation/Contribution/Index.rst]: Done.

**Review by ohader — APPROVED** [review]

**Comment by ohader** [comment]
TER foo done

**Pull request merged**